.github/frontend_build.yaml

```diff
@@ -0,0 +1,27 @@
+name: Node.js CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 14.x, 15.x]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - run: cd client
+    - run: npm i
+    - run: npm install
+    - run: npm start
```

Comment:
Don't do a `npm start`. Instead do `npm build` and see if the build is successful.

Reply:
Yeah I have done those changes, npm build does work!